NotesApp/Reusable/CustomAddButton.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  CustomAddButton.swift
+//  NotesApp
+//
+//  Created by Luka Gujejiani on 31.10.24.
+//
+
+import UIKit
+
+final class RoundedIconButton: UIButton {
+    // MARK: - Initializers
+    init(iconName: String, pointSize: CGFloat = 12, backgroundColor: UIColor = .darkGray, tintColor: UIColor = .white, cornerRadius: CGFloat = 25, size: CGFloat = 70) {
+        super.init(frame: .zero)
+        
+        setupButton(iconName: iconName, pointSize: pointSize, backgroundColor: backgroundColor, tintColor: tintColor, cornerRadius: cornerRadius, size: size)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Setup
+    private func setupButton(iconName: String, pointSize: CGFloat, backgroundColor: UIColor, tintColor: UIColor, cornerRadius: CGFloat, size: CGFloat) {
```

Comment:
when you have so many params you should use multiline arguments like this
private func setupButton(
iconName: String,
 pointSize: CGFloat,
 backgroundColor: UIColor, 
tintColor: UIColor, 
cornerRadius: CGFloat,
 size: CGFloat
) {

it makes code more readable an clean